database/migrations/2020_03_21_150221_make_gtad_group_names_unique.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class MakeGtadGroupNamesUnique extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('majors', static function (Blueprint $table): void {
+            $table->string('gtad_majorgroup_name')->unique()->change();
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     */
+    public function down(): void
+    {
+        Schema::table('geo', static function (Blueprint $table): void {
```

Comment:
Wrong table name? `geo`

Reply:
Whoops, I copy-pasted that from the internet